engine/api/action/aggregate.go

```diff
@@ -0,0 +1,233 @@
+package action
+
+import (
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/database/gorpmapping"
+	"github.com/ovh/cds/engine/api/group"
+	"github.com/ovh/cds/sdk"
+)
+
+type view []aggregator
+
+func (v view) Exec(db gorp.SqlExecutor, as ...*sdk.Action) error {
+	if len(as) > 0 {
+		for i := range v {
+			if err := v[i](db, as...); err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+type aggregator func(gorp.SqlExecutor, ...*sdk.Action) error
+
+// FullView for action of type default.
+var FullView = view{
```

Comment:
action.FullView.Exec is a bit verbose and meaningless.

I would prefer: `action.FullQuery.Exec(api.mustDB(),a)…`
or
`action.FullView(api.mustDB(),a)`